src/Quasar/QuasarF.purs

```diff
@@ -230,3 +233,11 @@ deleteMount
   → QuasarFE Unit
 deleteMount path =
   DeleteMount path id
+
+getMetastore ∷ QuasarFE (Metastore ())
+getMetastore = GetMetastore id
+
+putMetastore
+  ∷ { initialize ∷ Boolean, metastore ∷ Metastore (password ∷ String) }
+  → QuasarFE Unit
+putMetastore ms = PutMetastore ms id
```

Comment:
We should add the appropriate versions of these in `Quasar.Advanced.QuasarAF` too.